lib/generators/react_on_rails/templates/redux/base/client/app/lib/middlewares/loggerMiddleware.js

```diff
@@ -0,0 +1,23 @@
+/* eslint no-console: 0 */
+import lodash from 'lodash';
+
+export default function logger({ getState }) {
```

Comment:
// This logger should be configured not to run in a production environment.
// See https://github.com/petehunt/webpack-howto#6-feature-flags for how we might turn this off for production.